src/applications/vaos/components/calendar/CalendarRow.jsx

```diff
@@ -11,25 +12,25 @@ function isCellDisabled({ date, availableSlots, minDate, maxDate }) {
   if (
     (Array.isArray(availableSlots) &&
       !availableSlots.some(slot => slot?.start?.startsWith(date))) ||
-    moment(date).isBefore(moment().format('YYYY-MM-DD'))
+    isBefore(parseISO(date), new Date())
   ) {
     disabled = true;
   }
 
   // If minDate provided, disable dates before minDate
   if (
     minDate &&
-    moment(minDate).isValid() &&
-    moment(date).isBefore(moment(minDate))
```

Comment:
Hmm aren't minDate and maxDate passed in as Date objects? Why is parseISO needed? We should either update the propTypes or remove these parseISO calls.